data/hai/hai reveal 1 intro.txt

```diff
@@ -472,9 +472,17 @@ mission "Hai Tour [5] Tour Ends"
 			label "skipped tours"
 			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall. "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils. This unprecedented raid in broad daylight has the security forces scrambling to redistribute appropriate resources to handle the threat, and all security personnel on leave have been reactivated until better support can be provided."`
 			label "that's awful"
+			choice
+				`	Observe the reaction of the Hai in the room.`
+				`	Ignore the news and move on.`
```

Comment:
These actions should have brackets around them. 

I feel like changing the second option to (Move on.) or (Keep walking.) might be better. Ignoring the news appears apathetic, unless the player is a xenophobe.

Reply:
Could you please try the "suggest changes" feature? It'll make it easier to apply your changes (I can just press a button). To suggest a change, use the button that looks like  folded page with a + and - inside it.

Reply:
Okay, I wasn't aware of it. I'll try it for the next one.